src/dash/parser/DashParser.js

```diff
@@ -135,14 +135,81 @@ function DashParser(config) {
         return manifest;
     }
 
+    function processXml(data) {
+        const xml = cmlParseXml(data)
+        const root = xml.childNodes.find(child => child.nodeName === 'MPD');
+
+        function processNode(node) {
+            // Convert tag name
+            let p = node.nodeName.indexOf(':');
+            if (p !== -1) {
+                node.__prefix = node.prefix;
+                node.nodeName = node.localName;
+            }
+
+            const { childNodes, attributes, nodeName } = node;
+            node.tagName = nodeName;
+
+            // Convert attributes
+            for (let k in attributes) {
+                let value = attributes[k];
+
+                if (nodeName === 'S') {
+                    value = parseInt(value);
+                }
+                else {
+                    for (let i = 0, len = matchers.length; i < len; i++) {
+                        const matcher = matchers[i];
+                        if (matcher.test(nodeName, k, value)) {
+                            value = matcher.converter(value);
+                            break;
+                        }
+                    }
+                }
+
+                node[k] = value;
+            }
+
+            // Convert children
+            const len = childNodes?.length;
+
+            for (let i = 0; i < len; i++) {
+                const child = childNodes[i];
+
+                if (child.nodeName === '#text') {
+                    node.__text = child.nodeValue;
```

Comment:
This replaces `__text` rather than appending to it. Almost certainly not what's intended if there are multiple text children.

Reply:
This is a direct port of the old code

Reply:
https://github.com/Dash-Industry-Forum/dash.js/blob/d31387d2a0ca4f6fbe6a1df43c60de5b710bc2a9/externals/tXml.js#L230

Reply:
OK, it might also have been broken before!